gradle/github-release.gradle

```diff
@@ -0,0 +1,206 @@
+import groovy.text.GStringTemplateEngine
+import org.codehaus.groovy.runtime.DateGroovyMethods
+
+apply plugin: 'distribution'
```

Comment:
Previous custom github release code exists here. Nothing new.

Reply:
I might borrow this now that it's separate :P